pilot/lib/pilot/build_manager.rb

```diff
@@ -43,6 +43,10 @@ def upload(options)
 
       UI.success("Successfully uploaded the new binary to App Store Connect")
 
+      if not config[:skip_waiting_for_build_processing]
```

Comment:
```suggestion
      unless config[:skip_waiting_for_build_processing]
```

That should make rubocop happy re these two:

```
[16:42:10]: ▸ pilot/lib/pilot/build_manager.rb:46:7: C: Style/NegatedIf: Favor unless over if for negative conditions.
[16:42:10]: ▸ if not config[:skip_waiting_for_build_processing] ...
[16:42:10]: ▸ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[16:42:10]: ▸ pilot/lib/pilot/build_manager.rb:46:10: C: Style/Not: Use ! instead of not.
[16:42:10]: ▸ if not config[:skip_waiting_for_build_processing]
[16:42:10]: ▸ ^^^
```

Reply:
👍